src/Html/Attributes.php

```diff
@@ -4,17 +4,38 @@
 
 class Attributes
 {
+    /**
```

Comment:
Can you revert these?

Reply:
Oh, This commit for previous PR.
I revert this now.